app/src/main/java/org/helfoome/presentation/drawer/MyReviewActivity.kt

```diff
@@ -73,14 +55,29 @@ class MyReviewActivity : BindingActivity<ActivityMyReviewBinding>(R.layout.activ
         }
     }
 
-    private fun adapterClickListener(it: Int) {
-        when (it) {
-            ENLARGE -> {
-                startActivity<MainActivity>()
-            }
+    private fun startRestaurant() {
+        startActivity<MainActivity>()
+    }
+
+    private fun deleteReview(reviewId: String) {
+        val bind = DialogMyReviewDeleteBinding.inflate(LayoutInflater.from(this@MyReviewActivity))
```

Comment:
```suggestion
        val bind = DialogMyReviewDeleteBinding.inflate(layoutInflater)
```
액티비티에서 작성된 코드니까 그냥 해당액티비티의 레이아웃인플레이터를 갖고가서 쓰시면 됩니다!